src/DotLiquid/Hash.cs

```diff
@@ -174,18 +179,26 @@ protected virtual object GetValue(string key)
             return null;
         }
 
+        /// <summary>
+        /// Provides strongly-typed access to each of the keys in the Hash
+        /// </summary>
+        /// <typeparam name="T"> A generic parameter that specifies the return type of the column.</typeparam>

```

Comment:
nit: extraneous white space at start of comment
```suggestion
        /// <typeparam name="T">A generic parameter that specifies the return type of the column.</typeparam>
```